Chrono.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** Chrono.hpp
+*/
+
+#ifndef CHRONO_HPP
+    #define CHRONO_HPP
+
+    #include <iostream>
+    #include <chrono>
+
+class Chrono {
```

Comment:
can you add comments/documentation so that anyone can understand what's in the file without a lot of context?

Reply:
Could you check if that satisfies your request ?

Reply:
perfect, thx :+1: